lib/proxy/plugin_initializer.rb

```diff
@@ -320,7 +320,7 @@ def load_classes
   end
 
   def validate_settings(plugin, config)
-    result = execute_validators(plugin.plugin_default_settings.keys.map {|k| {:name => :presence, :setting => k}}, config)
+    result = execute_validators(plugin.plugin_default_settings.keys.map {|k| {:name => :presence_allow_empty, :setting => k}}, config)
```

Comment:
I can't quite see the implications of this change. @witlessbird?

Reply:
I do think that changing the default here for every plugin default can open us up to unexpected errors. Not all code might be able to deal with it.

Reply:
It only allows empty string, not `nil`.

Reply:
Still, changing this for every plugin sounds incorrect. Changes should be as small as possible and I can't tell exactly what this change will impact.

Reply:
I am not sure how code needs to be ready to deal with empty string, you can set it to "x" or some UTF-8 emoticon character today and the same code would need to work it out. Allowing nil that's another story, but that's not what I do in this PR.

Reply:
Yet, without this change the test pass which makes me think we shouldn't change this. Could we remove the PresenceAllowEmpty part of this PR?